README.md

```diff
@@ -120,6 +120,21 @@ In the Run configurations, copy the PHPUnit configuration and use these settings
 2. right-click on `Tests/Acceptance/StarterCest.php`
 3. Run 'Acceptance (Codeception)'
 
+## Developing locally
+
+In order to work on the extension locally, you can use a local environment (local PHP, server) or use
+a widely adopted tool in TYPO3 Community, [ddev](https://github.com/drud/ddev), which we recommend.
+
+## Running Composer commands in the DDEV container
+
+If you use ddev, then you can use the provided command.
+
+Example:
+
+```bash
+ddev composer ci:ts:lint
+```
+
 ## Security
 
 Libraries and extensions do not need the security check as they should not have
```

Comment:
After I tried to do a `ddev start` (which obviously is not the right command), I had the following untracked files/directories in my project directory:

```
        .ddev/.dbimageBuild/
        .ddev/.ddev-docker-compose-base.yaml
        .ddev/.ddev-docker-compose-full.yaml
        .ddev/.global_commands/
        .ddev/.homeadditions/
        .ddev/.sshimageBuild/
        .ddev/.webimageBuild/
        .ddev/apache/
        .ddev/commands/
        .ddev/homeadditions/
        .ddev/nginx_full/
        .ddev/providers/
```

I think we need to add those to the `.gitignore` and allowlist `.ddev/config.yaml`.

Reply:
I have .gitignore file in .ddev directory that is automatically generated by ddev on start and has following content

```
#ddev-generated: Automatically generated ddev .gitignore.
# You can remove the above line if you want to edit and maintain this file yourself.
/.gitignore

/**/*.example
/.dbimageBuild
/.dbimageExtra
/.ddev-docker-*.yaml
/.*downloads
/.global_commands
/.homeadditions
/.sshimageBuild
/.webimageBuild
/.webimageExtra
/apache/apache-site.conf
/commands/.gitattributes
/commands/db/mysql
/commands/host/launch
/commands/web/live
/commands/web/xdebug
/config.*.y*ml
/db_snapshots
/import-db
/import.yaml
/nginx_full/nginx-site.conf
/sequelpro.spf
/**/README.*

```

Reply:
Are you sure? I have no untracked files in .ddev directory, as there is .gitignore in it generated by ddev.